ajax/libs/mapbox-gl/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "mapbox-gl",
+  "description": "A WebGL interactive maps library",
+  "version": "0.32.1",
+  "main": "src/index.js",
+  "filename": "src/index.min.js",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mapbox/mapbox-gl-js.git"
+  }, 
+  "npmName": "mapbox-gl",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "dist/mapbox-gl.css",
+        "src/index.js"
+      ]
```

Comment:
@maruilian11 I think the plugins ([listed here](https://www.mapbox.com/mapbox-gl-js/plugins/)) should be a separate pull request, as they are separate packages available on npm (e.g. [mapbox-gl-draw](https://www.npmjs.com/package/@mapbox/mapbox-gl-draw)).